btr-api/tests/unit/resources/test_requests.py

```diff
@@ -164,7 +173,7 @@ def test_put(app, client, session, jwt, requests_mock, sample_user, test_name, u
 
 @pytest.mark.parametrize(
     'test_name, user_type',
-    [('test_put_no_auth', UserType.USER_COMPETENT_AUTHORITY)],
+    [('test_put_no_auth', UserType.USER_PUBLIC)],
```

Comment:
makes sense to have the USER_COMPETENT_AUTHORITY test case here too doesn't it?

Reply:
not for no auth